src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -145,3 +148,140 @@
     login_theme="keycloak",
     opts=resource_options,
 )
+
+# Create OL Public Realm
+ol_apps_realm = keycloak.Realm(
+    "olapps",
+    access_code_lifespan="30m",
+    access_code_lifespan_user_action="15m",
+    attributes={
+        "business_unit": f"operations-{env_name}",
+    },
+    display_name="OLApps",
+    display_name_html="<b>MIT Open Learning Apps</b>",
+    enabled=True,
+    login_theme="base",
+    duplicate_emails_allowed=False,
+    otp_policy=keycloak.RealmOtpPolicyArgs(
+        algorithm="HmacSHA256",
+        digits=6,
+        initial_counter=2,
+        look_ahead_window=1,
+        period=30,
+        type="totp",
+    ),
+    realm="olapps",
+    reset_password_allowed=True,
+    verify_email=True,
+    password_policy="upperCase(2) and digits(2) and length(12) and specialChars(2) and forceExpiredPasswordChange(365) and notUsername and notEmail",  # noqa: E501,S106 # pragma: allowlist secret
```

Comment:
I'm not sure that we want to force end-users to rotate their passwords anually. Maybe also change the requirements for uppercase/digits/special characters to 1? And maybe require at least 2 of the 3?